riverdriver/riverdatabasesql/river_database_sql_driver.go

```diff
@@ -873,6 +901,12 @@ func (t *ExecutorSubTx) Begin(ctx context.Context) (riverdriver.ExecutorTx, erro
 	if err != nil {
 		return nil, err
 	}
+
+	_, err = t.Exec(ctx, "SET search_path TO 'public'")
+	if err != nil {
+		return nil, err
+	}
```

Comment:
Hmm, an added round trip for every use here? That doesn't seem good, any way we can avoid it?

Reply:
Oops, this was a mistake. I took it back out again.